src/Engine/Renderer/Renderers/DebugRender.hpp

```diff
@@ -38,7 +39,7 @@ class RA_ENGINE_API DebugRender final
     void addPoint( const Core::Vector3& p, const Core::Utils::Color& color );
```

Comment:
can you add some documentation?

Reply:
Well, the update here are from the update of the class names. 
So nothing new, and I'm not aware of what theses methods do.